class/practice7.1/MaybeTheBestCodeEver.cpp

```diff
@@ -0,0 +1,56 @@
+#include <iostream>
+#include <print>
+#include "Rectangle.hpp"
+
+int main() {
+    double widths[5] = {0.0};
```

Comment:
there is no need to have these 2 arrays. Lets use vector of Rect objects. Then logic to get biggest area can be done with C++ standard library algorithms